net2/HostManager.js

```diff
@@ -802,6 +802,11 @@ module.exports = class HostManager {
     const versionUpdate = await sysManager.getVersionUpdate();
     if (versionUpdate)
       json.versionUpdate = versionUpdate;
+    if (require('fs').existsSync("/home/pi/.firewalla/config/.no_auto_upgrade")) {
```

Comment:
* should not use existsSync
* also no need to check every time when loading init data
* follow SysInfo.js to check the file periodically.